### DISCUSSION TITLE
Can't log in

### DISCUSSION THREAD

**Discussion by dolianich** [vercel/community#2822] [Help] [answered]
### Summary

Hello, this error shows up after I passed my phone number at the verification window
<img width="453" alt="Снимок экрана 2023-06-12 в 01 50 45" src="https://github.com/vercel/community/assets/69437697/f5a8022e-c581-4754-9954-282cee48b080">

### Steps to Reproduce

Maybe it's needed to change the phone number somehow or disconnect it, so I could sign up again

**Comment by swarnava** [comment, marked as answer]
Please email [registration@vercel.com](mailto:registration@vercel.com) and include the email and username used for account creation. To protect against platform abuse, Vercel employs multiple layers of security and verification. Providing additional information about your account and the project you are deploying can help accelerate this process.
